### PR TITLE
[API] Add an endpoint to show all products

### PR DESCRIPTION
**Pull request by cenuse-dan**

### CODE REVIEW

**Review by hasdo113 — APPROVED** [review]
Ai facut bine!

File: src/main/java/ro/unibuc/hello/service/ProductService.java [hunk 1]
```diff
@@ -18,6 +23,23 @@ public void addProduct(ProductDTO productDTO) {
...
+        products.forEach(product -> {
+            productsDto.add(ProductDTO.builder().productName(product.getProductName())
+                    .productDescription(product.getProductDescription())
+                    .brandName(product.getBrandName())
+                    .price(product.getPrice())
+                    //.categoryDTO(CategoryDTO.builder().categoryName(product.getCategory().getName()).build())
+                    .stock(product.getStock()).build());
+
+        });
+        System.out.println(productsDto);
```
Comment (ralucapascu) [inline src/main/java/ro/unibuc/hello/service/ProductService.java:38]: Not needed

Comment (hasdo113) [reply src/main/java/ro/unibuc/hello/service/ProductService.java:38]: Si spatiul ala extra e rau rau

**Review by ralucapascu — APPROVED** [review]

**Review by hasdo113 — APPROVED** [review]
Ai facut bine!

**Pull request merged**